modules/ROOT/pages/kafka/kafka-connector-examples.adoc

```diff
@@ -2,54 +2,78 @@
+. Set the *Display Name* of *Listener* to `Push message endpoint` and the *Path* to `/pushMessage`.
+. Drag a *Logger* component to the right of *Push message endpoint* on the Studio canvas and set the message to:
 +
 [source,xml]
 ----
-#['New message arrived: ' ++ payload ++ &quot;, key:&quot; ++ attributes.key ++ &quot;, partition:&quot; ++ attributes.partition ++ &quot;, offset:&quot; ++ attributes.offset ]
+#[&quot;Message: '&quot; ++ payload.message ++ &quot;' is going to be published to topic: '&quot; ++ payload.topic ++ &quot;'.&quot;]
 ----
 +
-The Consumer flow appears in Studio as:
+. Drag the Apache Kafka *Publish* operation to the right of *Logger* on the Studio canvas.
+. Set the *Display Name* field to `Producer`, the *Topic* field to `#[payload.topic]`, and the *Key* field to `#[now()]`.
+. Click the green plus icon to the right of the *Connector configuration* field to access the global element configuration fields.
+.. In the *Bootstrap server URLs* field, select *Edit inline* and then click the green plus sign.
+.. Enter the value `${config.basic.bootstrapServers}` and click *Finish*.
+. Declare and complete the variables previously described in the corresponding YAML file (for example: `src/main/resources/properties/kafka-producer.yaml`), and load that properties file as a global element.
 +
-image::kafka/kafka-consumer-studio-flow.png[]
+For more information about using a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders]
 +
-. In the next flow, drag *HTTP Listener* to the canvas, set the *Path* to `/pushMessage`, and use the defaults for the configuration.
-. Drag *Logger* to the canvas and set the message to:
+*Is the previous line correct and in the right place for the procedure?*
 +
-[source,xml]
-----
-#[&quot;Message: '&quot; ++ payload.message ++ &quot;' is going to be published to topic: '&quot; ++ payload.topic ++ &quot;'.&quot;]
-----
+. Drag the *Set Payload* component to the right of *Producer* on the Studio canvas.
+. Change the *Display Name* field to `Push response builder`.
+. Set the *Value* to `Message successfully sent to Kafka topic.`
+
+=== Create the Consumer Flow
+
+The Consumer flow consumes a message from Kafka.The following screenshot shows the Consumer flow in Studio:
+
+.Create a flow like this one to retrieve a message from Kafka.
+image::kafka/kafka-consumer-studio-flow.png[Kafka Connector Consumer flow in Studio]
+
+To create the Consumer flow:
+
+. Drag the Apache Kafka *Consume Message* input source to the Anypoint Studio canvas.
+. Change the flow name to `Consumer-Flow`.
+. Set the *Display Name* field of the *Message Listener* to `Consume message endpoint`.
+. Click the green plus icon to the right of the *Connector configuration* field to access the global element configuration fields.
+. Complete these fields:
+.. In the *Bootstrap server URLs* field, select *Edit inline* and then *+*.
```

Comment:
```suggestion
.. In the *Bootstrap server URLs* field, select *Edit inline* and then click *+*.
```
select? Should we be using this instead of "click the green plus sign"?